### PR TITLE
support fetching fargate prices and reporting fargate nodes

### PR DESCRIPTION
**Pull request by tzneal**
*Issue #, if available:*

Fargate Pricing - #35 
Show another digit of precision for prices - #87

*Description of changes:*

Support fargate nodes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by mbevc1 — APPROVED** [review]
Awesome, thanks :+1:

File: cmd/eks-node-viewer/main.go [hunk 1]
```diff
@@ -123,6 +123,11 @@ func startMonitor(ctx context.Context, settings *monitorSettings) {
 				p := obj.(*v1.Pod)
 				if !isTerminalPod(p) {
 					cluster.AddPod(model.NewPod(p), settings.pricing)
+					node, ok := cluster.GetNode(p.Spec.NodeName)
+					// need to potentially update node price as we need the fargate pod in order to figure out the cost
+					if ok && node.IsFargate() && node.Price != node.Price {
```
Comment (bwagner5) [inline cmd/eks-node-viewer/main.go:128]: is `node.Price != node.Price` right? Seems like that should always be false?

**Review by bwagner5 — COMMENTED** [review]
one question, otherwise LGTM

File: cmd/eks-node-viewer/main.go (hunk 1 above)
Comment (tzneal) [reply cmd/eks-node-viewer/main.go:128]: I use NaN to represent an unknown price, and NaN != NaN. Probably not the clearest choice I ever made 😆

Comment (tzneal) [reply cmd/eks-node-viewer/main.go:128]: https://go.dev/play/p/rwq-A_4qDPm

**Pull request merged**